frontend/src/i18/english.ts

```diff
@@ -249,5 +249,27 @@ export const english = {
     'Change address': 'Change address',
     'verify email before your journey':
       'Before your journey in Visdom, we need to verify your email address.',
+    'Join project': 'Join project',
+    'Join project info 1/2':
+      'You have been invited to join <strong>{{name}}</strong> as <strong>{{type}}</strong>',
+    'Join project info 2/2': 'Join project now?',
+    'Cancel joining project': 'Cancel joining project',
+    'Cancel join project warning':
+      'Are you sure you want to cancel joining this project?',
+    'New project added': 'New project added',
+    'New project added description':
+      'Awesome! <strong>{{name}}</strong> has been added in your projects.',
+    'No, thanks': 'No, thanks',
+    'Project link unvalid': 'Project link unvalid',
```

Comment:
```suggestion
    'Project link invalid': 'Project link invalid',
```

Shouldn't this be "invalid" not "unvalid"?

same in component naming